lib/gollum/views/layout.rb

```diff
@@ -20,10 +20,18 @@ def has_path
         !@path.nil?
       end
 
+      def page_dir
+        @page_dir
+      end
+
       def base_url
         @base_url
       end
 
+      def custom_path
+        @base_url + (@page_dir.nil? ? '' : '/') + @page_dir
```

Comment:
I think we want to encourage using string interpolation wherever possible:
`"#{@base_url}#{@page_dir.nil? ? '' : '/'}#{@page_dir}"`